spec/request/swagger_spec.rb

```diff
@@ -1278,6 +1278,18 @@
 
     describe 'facility locator tests' do
       context 'successful calls' do
+        let(:provider) { FactoryBot.build(:provider, :from_provider_info) }
+        let(:provider_services_response) do
+          {
+            'CareSiteAddressStreet' => Faker::Address.street_address,
+            'CareSiteAddressCity' => Faker::Address.city,
+            'CareSiteAddressZipCode' => Faker::Address.zip,
+            'CareSiteAddressState' => Faker::Address.state_abbr,
+            'Latitude' => Faker::Address.latitude,
+            'Longitude' => Faker::Address.longitude
```

Comment:
I'd recommend hardcoding. Not sure fake info here brings much value

Reply:
ok, I'll make the change.